rust/ql/lib/utils/test/internal/InlineExpectationsTestImpl.qll

```diff
@@ -4,6 +4,8 @@ private import codeql.util.test.InlineExpectationsTest
 
 module Impl implements InlineExpectationsTestSig {
   class ExpectationComment extends R::Comment {
+    ExpectationComment() { this.fromSource() }
```

Comment:
There was a comment inside the standard library starting with `$target`, so I had to limit expectations comments to those inside the source, which seems right in any case.